src/MahApps.Metro/Themes/SplitView.xaml

```diff
@@ -8,6 +8,7 @@
         <Setter Property="OpenPaneLength" Value="320" />
         <Setter Property="PaneBackground" Value="{DynamicResource {x:Static SystemColors.ControlLightBrushKey}}" />
         <Setter Property="PaneForeground" Value="{DynamicResource MahApps.Brushes.Text}" />
+        <Setter Property="OverlayBrush" Value="{DynamicResource MahApps.Brushes.Window.FlyoutOverlay}" />
```

Comment:
@timunie I think we should leave it null, because the old Hamburger doesn’t had this.

Reply:
I am ok with transparent. I think if we leave it null, it will not work anymore, or am I wrong?

Reply:
@punker76 just as an inforamtion: It was `Transparent` before my changes and not `{x:null}`